<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreGridTile</class>
 <widget class="QWidget" name="StoreGridTile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>221</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgb(237, 237, 237);</string>
  </property>
  <layout class="QVBoxLayout" name="widgetLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="contentRoot">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="contentRootLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="titleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">Title</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="oldPriceLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <strikeout>true</strikeout>
         </font>
        </property>
        <property name="text">
         <string notr="true">Old price</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" rowspan="2">
       <widget class="QPushButton" name="addToCartButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgb(180, 230, 60), stop:1 rgb(140, 202, 39));</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/cart-plus.svg</normaloff>:/icons/cart-plus.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QLabel" name="discountLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white; 
      background-color: rgb(110, 29, 114); 
      border-radius: 3px;
      padding: 10px;</string>
        </property>
        <property name="text">
         <string notr="true">Discount</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="priceLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">Price</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QWidget" name="coverHolder" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>255</width>
          <height>131</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>255</width>
          <height>131</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #d9d9d9;</string>
        </property>
        <widget class="QLabel" name="coverLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>13</y>
           <width>255</width>
           <height>118</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>255</width>
           <height>118</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>255</width>
           <height>118</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(237, 237, 237);</string>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
        <widget class="QLabel" name="ownedLabel">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(237, 237, 237); border: 1px solid rgba(64, 64, 64, 0.15); color: rgb(64, 64, 64); padding: 8px; padding-bottom: 6px;</string>
         </property>
         <property name="text">
          <string>IN LIBRARY</string>
         </property>
        </widget>
        <widget class="QLabel" name="wishlistedLabel">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>0</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(255, 241, 219); border: 1px solid rgba(99, 65, 9, 0.15); color: rgb(99, 65, 9); padding: 8px; padding-bottom: 6px;</string>
         </property>
         <property name="text">
          <string>WISHLISTED</string>
         </property>
        </widget>
        <widget class="QLabel" name="upcomingLabel">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>0</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(230, 247, 254); border: 1px solid rgba(1, 17, 25, 0.15); color: rgb(1, 17, 25); padding: 8px; padding-bottom: 6px;</string>
         </property>
         <property name="text">
          <string>SOON</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
